src/net_processing.cpp

```diff
@@ -2566,6 +2572,18 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         return;
     }
 
+    if (msg_type == NetMsgType::BLOCKRELAY) {
```

Comment:
Wouldn't "BLKRELAYONLY" be a better name for this concept, or even separating it out into its components as "TXRELAY false" and "ADDRRELAY false" ? Seems surprising that a "BLOCKRELAY" message would change tx and addr relay behaviour, but not change block relay behaviour...

Reply:
How about `disabletx`?

Reply:
Sure, `disabletx` or `txrelay false` sound about the same to me; though opting-out vs opting-in still maybe seems odd?

Reply:
Well my thinking is that the default on the network is that everything gets relayed, so the natural thing to add is an opt-out.